spec/spec_acceptance_helper.rb

```diff
@@ -0,0 +1,24 @@
+require 'beaker-rspec'
```

Comment:
This file should not be here. Looks like it was renamed but the old one was not removed.